tests/gnrc_sock_dodtls/README.md

```diff
@@ -0,0 +1,36 @@
+# Overview
+
+This folder contains a test application for RIOT's sock-based DNS over DTLS
+client.
+
+# How to test with native
+
+Setup up a tap interface:
+
+```console
+$ sudo ip tuntap add dev tap0 mode tap user $(id -u -n)
+$ sudo ip a a 2001:db8::1/64 dev tap0
+$ sudo ip link set up dev tap0
+```
+
+TBD: add how to start DNS over DTLS server here
+
+Then run the test application
+
+```console
+make term
+```
+
+Now use the RIOT shell to configure the DNS over DTLS server and request
+`example.org` from it
+
+```console
+> dodtls server [2001:db8::1]:853 5853 Client_identifier secretPSK
```

Comment:
What is `Client_identifier`?
Can `secretPSK` be just any string?

Reply:
You wouldn't ask those questions if you had used DTLS with PSK already ;-). But will clarify.